components/match2/commons/match_group_input_util.lua

```diff
@@ -205,29 +205,33 @@ function MatchGroupInputUtil.readOpponent(match, opponentIndex, options)
 	opponent.name = Opponent.toName(opponent)
 
 	local substitutions
-	if opponent.type == Opponent.team and Logic.isNotEmpty(opponent.name) then
+	if opponent.type == Opponent.team then
 		local manualPlayersInput = MatchGroupInputUtil.extractManualPlayersInput(match, opponentIndex, opponentInput)
 		substitutions = manualPlayersInput.substitutions
 		--a variation of `MatchGroupInput.readPlayersOfTeam` that returns a player array
 		opponent.players = MatchGroupInputUtil.readPlayersOfTeamNew(
-			opponent.name,
+			Opponent.toName(opponent),
```

Comment:
Is this change needed?

Just before the condition, this is already done?

Reply:
fucked up the merge conflict resolve ...